GoogleForCreators/web-stories-wp
# Template Updates: DIY

## Context

Updating the templates to match [new designs](https://www.figma.com/file/NHHMwIfxMnz39NxqkrFb7R/Stories-Post-Stable?node-id=7357%3A545144).

~Note: new animations are being added later, so those have not been implemented here.~ 
April 10, 2021: Custom animations have been removed for existing animations 

## Summary

Updates the DIY template.

## Relevant Technical Choices

n/a

## To-do


## User-facing changes

**Before:**

https://user-images.githubusercontent.com/22185279/113633370-1466b400-962a-11eb-97fd-0fe825f8295b.mov


**After:**

https://user-images.githubusercontent.com/22185279/113633367-12045a00-962a-11eb-8521-e6d3224c166f.mov

Should match these specifications:
![image](https://user-images.githubusercontent.com/22185279/113633440-2fd1bf00-962a-11eb-88b6-6e6820bff1f1.png)


## Testing Instructions

1. Select the template the `Explore Templates` section in the dashboard
2. Go to the preview
3. Watch the new template with animations
4. Make sure the colors and animations line up with the figma document - there should be no custom animations 

### QA






This PR can be tested by following these steps:
1. Select the template the `Explore Templates` section in the dashboard
2. Go to the preview
3. Watch the new template with animations

### UAT






## Reviews

### Does this PR have a security-related impact?

no

### Does this PR change what data or activity we track or use?

no

### Does this PR have a legal-related impact?

no

## Checklist




---



Fixes #6971
